docs/integrations/amazon-aws/cloud-infrastructure-security-for-aws.md

```diff
@@ -79,7 +79,7 @@ Cloud Infrastructure Security for AWS utilizes the following log types:
 }
 ```
 
-## Sample query
+## Sample queries
```

Comment:
Similar question as above. Why would you change "Sample query" to "Sample queries" when there is only one query shown?

Reply:
I was hoping we could treat those headers as UI (they appear in every single apps doc) and keep them uniform. Here's an example -- these docs say "Endpoints" even if there's only one.

<img width="661" alt="Screenshot 2024-01-23 at 5 13 27 PM" src="https://github.com/SumoLogic/sumologic-documentation/assets/56411016/400843f1-fd02-4f3d-94f8-1fef6c0e7b63">

Reply:
Makes sense to me, @kimsauce.